hw/ip/kmac/doc/_index.md

```diff
@@ -57,10 +57,10 @@ It gives some freedome to the software not requiring to align the messages.
 It also doesn't need the message length information.
 
 The fed messages go into the KMAC core regardless of KMAC enabled or not.
-But the KMAC core just forwards the messages to SHA3 core in case of KMAC disabled.
+The KMAC core forwards the messages to SHA3 core in case of KMAC disabled.
 KMAC core prepends the encoded secret key as described in the SHA3 Derived Functions specification.
 It is expected that the software writes the encoded output length at the end of the message.
-For KDF triggered by KeyMgr, the encoded output length is appended inside the KeyMgr interface module in KMAC HWIP.
+For AppIntf triggered KMAC operations, the encoded output length is appended inside the AppIntf module in the KMAC HWIP.
```

Comment:
nit: maybe "For hashing operations triggered by an IP through the application interface, the encoded output length is appended inside the AppIntf module in the KMAC HWIP" ?